iis-wf/iis-wf-referenceextraction/src/main/java/eu/dnetlib/iis/wf/referenceextraction/softwareurl/CachedWebCrawlerJob.java

```diff
@@ -1,40 +1,37 @@
 package eu.dnetlib.iis.wf.referenceextraction.softwareurl;
 
+import org.apache.hadoop.conf.Configuration;
```

Comment:
General note: refactorization of this job is also quite significant and it could be also rewritten into Spark 2 with dataframes.

Reply:
Agree. Let's close this round and I will align both `PatentMetadataRetrieverJob` and `CachedWebCrawlerJob` to rely on dataframes as a separate task. Just to keep this PR more compact.

I have created dedicated ticket for this task: #1130.